config/routes.rb

```diff
@@ -7,6 +7,7 @@
 
   resource :contractor, only: [:create, :show, :edit, :update] do 
     resources :projects, only: [:create, :show, :index] do
+      delete 'remove_dev', to: "projects#remove_dev"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.